app/src/main/java/com/abominableshrine/taptounlock/TapPatternDetectorClient.java

```diff
@@ -0,0 +1,69 @@
+package com.abominableshrine.taptounlock;
+
+import android.os.IBinder;
+
+/**
+ * Helper class to communicate with the TapPatternDetectorService
+ * <p/>
+ * This class intends to provide simple interfaces for clients without going through the hassle of
+ * Messaging.
+ */
+public abstract class TapPatternDetectorClient {
```

Comment:
@psycholold please use this class to interact with the service. I will implement it soon until then it should be easily mockable :-)